lib/components/common/Info.tsx

```diff
@@ -108,17 +108,18 @@ const styles = {
     }
 };
 
-const Info = (props: { setIsInfoOpen: (b: boolean) => void }) => {
-    const {setIsInfoOpen} = props
+const Info = (props: { setIsInfoOpen: (b: boolean) => void, onClose?: () => void }) => {
+    const {setIsInfoOpen, onClose = () => {}} = props
```

Comment:
issue (blocking): I think you missed the point of [this comment](https://github.com/MetaCell/cde-mapper/pull/22#discussion_r1516211515). I think we don't need to have both setIsInfoOpen and onClose, since both are called in the same circumstances. 
The component that uses the Info component should pass an onClose that does both things